packages/app/src/request.ts

```diff
@@ -40,9 +43,8 @@ export const getProtocol = (req: Request): Protocol => {
 
 export const getHostname = (req: Request): string | undefined => {
   let host: string = req.get('X-Forwarded-Host') as string
-
+  /* c8 ignore next */
```

Comment:
I think we should test in environment when there's no such header defined and address can't be trusted

Reply:
[`trustRemoteAddress` always returns a fn](https://github.com/tinyhttp/tinyhttp/blob/c5d4a90b3c2cf8ded8c5c1f281dd719ebf8cfcc5/packages/proxy-addr/src/index.ts#L69) and hence it is an unreachable branch

Reply:
let's eliminate the unreachable code then